src/systems/material.js

```diff
@@ -144,6 +145,16 @@ module.exports.System = registerSystem('material', {
     texture.minFilter = THREE.LinearFilter;
     setTextureProperties(texture, data);
 
+    // if we're on iOS, and the video is HLS, we currently need to do some hacks
+    if (utils.device.isIOS() && isHLS(videoEl)) {
```

Comment:
have you tested this on [Edge](https://webvr.rocks/microsoft_edge) (which supports WebVR in release channel)? Edge [supports HLS](http://caniuse.com/#feat=http-live-streaming) too; it may or may not have the same bugs.